feat: python3, PyYAML, and fix for older output method

yardbirdsax opened the pull request:
* Updates the Action to use the newer method of setting output values, since the older `set-ouput` command has been deprecated.
* Adds python and the PyYAML library to the base image, which may be useful for folks doing unusual things with Terraform external data sources.

yardbirdsax committed:
chore: update to newer output method

This updates the Action to use the newer syntax for providing output values.

yardbirdsax committed:
Merge 6b0dcca374f7e3a9ed6eff030f7e336620127477 into 0a99e9d381a986dfcdde0987e7c897d4bcf5be34

yardbirdsax committed:
auto-commit action.yml changes

nmische reviewed (approved):
Looks reasonable to me.

yardbirdsax committed:
feat: added Python3 and PyYAML to image

This adds Python3 and the PyYAML library to the base image, which may be useful for folks doing unusual things with Terraform external data sources.

yardbirdsax committed:
Merge 1c171afc6261240d523a5e3c4f7916c1cb519f2c into 0a99e9d381a986dfcdde0987e7c897d4bcf5be34

yardbirdsax committed:
auto-commit action.yml changes
